### ISSUE TITLE
  TOML parse error at line 1, column 1       |     1 | # INTRO       | ^     missing field `format_version`

### ISSUE THREAD

**Issue by Jack-Daly-Eng025** [rust-lang/rustlings#2069]
Using the most recent version of Rustlings as it's great for practicing and keeping up to date with Rust's version changes  but info.toml gives grief when you try to engage watch mode right out of the box now. 

Seems to have some whitespacing/syntax problem, I am looking in to resolving it myself but as far as I can tell, this line is commented correctly and as it's the very first, nothing is proceeding to that could to my imagination cause a parse error.

I will close the ticket if I manage to correct whatever is happening but I am wondering if anyone else in the community is running into a similar error today? 

Error Message is below for anyone curious. I could just be missing something simple. 
****************************************************************************************************************************************************8
Error: Failed to parse the `info.toml` file

Caused by:
    TOML parse error at line 1, column 1
      |
    1 | # INTRO
      | ^
    missing field `format_version`

*****************************************************************************************************************************************************8

**Comment by mo8it** [comment]
Are you using the new version of Rustlings on exercises that you started some time ago?
Version 6 of Rustlings is not compatible with older versions. So you need to start over with the exercises to use the new version. Otherwise, you need to stick to the old version.

**Comment by Jack-Daly-Eng025** [comment]
Nope, I updated to the latest 1.80.00 before hand. I do think VSCode may be playing a part but I do have Even Better Toml enabled, so I'm scratching my head a little.

**Comment by mo8it** [comment]
I don't mean the Rust version. I mean the version of the Rustlings exercises. Are you trying to run `rustlings` on a directory initialized using `rustlings init`?

Could you please share the output of `ls -l`?

**Comment by Jack-Daly-Eng025** [comment]
Sure, that's no bother. I'll try and uninstall/reinstall, just in the event it's just something that I may have triggered in error. 

The output of ls -l is below, I just removed my username.
total 276
-rw-rw-r--  1 username001 username001 86292 Jun 11 12:28 AUTHORS.md
-rw-rw-r--  1 username001 username001 25693 Jun 11 12:28 Cargo.lock
-rw-rw-r--  1 username001 username001   629 Jun 11 12:28 Cargo.toml
-rw-rw-r--  1 username001 username001 48488 Jun 11 12:28 CHANGELOG.md
-rw-rw-r--  1 username001 username001  4171 Jun 11 12:28 CONTRIBUTING.md
drwxrwxr-x 26 username001 username001  4096 Jun 11 12:28 exercises
-rw-rw-r--  1 username001 username001  1984 Jun 11 12:28 flake.lock
-rw-rw-r--  1 username001 username001  2338 Jun 11 12:28 flake.nix
-rw-rw-r--  1 username001 username001 42570 Aug  6 13:26 info.toml
-rw-rw-r--  1 username001 username001  3045 Jun 11 12:28 install.ps1
-rwxrwxr-x  1 username001 username001  4662 Jun 11 12:28 install.sh
-rw-rw-r--  1 username001 username001  1095 Jun 11 12:28 LICENSE
-rw-rw-r--  1 username001 username001  750 Jun 11 12:28 oranda.json
-rw-rw-r--  1 username001 username001  6349 Jun 11 12:28 README.md
-rw-rw-r--  1 username001 username001   278 Jun 11 12:28 shell.nix
drwxrwxr-x  2 username001 username001  4096 Jun 11 12:28 src
drwxrwxr-x  5 username001 username001 4096 Jun 11 12:37 target
drwxrwxr-x  3 username001 username001  4096 Jun 11 12:28 tests

**Comment by mo8it** [comment]
As I guessed, you are trying to run Rustlings on exercises that you started in June. You need to either stick to the old version of the Rustlings program or start over with the exercises and lose your progress.

To get back the old version of the Rustlings program, run `cargo install --force --path .` in this directory.

To start over with the new Rustlings program, delete this directory and run `rustlings init` to create a new one.

**Comment by Jack-Daly-Eng025** [comment]
Yeah I have it now. I changed directories and started a new folder. That seems to have fixed it. Cheers mate. Have a great day.
